integration-tests/src/test/java/org/apache/fineract/integrationtests/investor/externalassetowner/InitiateExternalAssetOwnerTransferTest.java

```diff
@@ -114,17 +115,16 @@ public void saleActiveLoanToExternalAssetOwner() {
             assertEquals(responseMap.get("resourceExternalId"), transferExternalId);
 
             String retrieveResponse = externalAssetOwnerHelper.retrieveTransferByLoanId(loanID.longValue());
-            Type retrieveType = new TypeToken<List<Map<String, Object>>>() {}.getType();
-            List<Map<String, Object>> retrieveResponseMap = new Gson().fromJson(retrieveResponse, retrieveType);
-            assertEquals(1, retrieveResponseMap.size());
-            assertEquals(retrieveResponseMap.get(0).get("transferExternalId"), transferExternalId);
-            assertEquals(retrieveResponseMap.get(0).get("status"), "PENDING");
+            Type retrieveType = new TypeToken<Page<Map<String, Object>>>() {}.getType();
+            Page<Map<String, Object>> retrieveResponseMap = new Gson().fromJson(retrieveResponse, retrieveType);
```

Comment:
I think we need an additional test case to verify that paging is working. Don't you think?